src/cloud/project/project-patch.md

```diff
@@ -6,22 +6,24 @@ functional_areas:
   - Upgrade
 ---
 The [Magento Cloud Patches](https://github.com/magento/magento-cloud-patches) package
-provides Magento Cloud patches, which improve the integration of all {{site.data.var.ee}} versions with Cloud environments and support quick delivery of critical fixes.
+delivers Magento patches and hot fixes which improve the integration of all `{{site.data.var.ee}}` versions with Cloud environments and supports quick delivery of critical fixes. It also delivers [custom patches]({{ site.baseurl }}/guides/v2.3/comp-mgr/patching.html#custom-patches) that you add to your {{site.data.var.ece}} project.
 
-The {{ site.data.var.mcp }} package is a dependency for the {{site.data.var.ct}} package. When you install or update the {{ site.data.var.ct }} package version, the Patches package is also installed or updated. You can also use and manage the {{ site.data.var.mcp }} as a stand-alone package for an existing {{ site.data.var.ece }} project.
+The `{{site.data.var.mcp}}` package is a dependency for the `{{site.data.var.ct}}` package and is installed or updated when you install or update `{{site.data.var.ct}}`. You can also use `{{site.data.var.mcp}}` as a stand-alone package to apply patches to an existing {{site.data.var.ee}} project.
 
-You can also use {{site.data.var.mcp}} to apply [custom patches]({{ site.baseurl }}/guides/v2.3/comp-mgr/patching.html#custom-patches) provided by support or third-party extension developers.  To use this feature,  copy the custom patch to the `/m2-hotfixes` directory in the {{ site.data.var.ee }} project root directory. Then, test the patch on your local workstation.
+When you deploy changes to the remote environment, `{{site.data.var.ct}}` uses `{{site.data.var.mcp}}` to check for pending patches and applies them automatically in the following order:
 
-{% include cloud/note-upgrade.md %}
+1. Apply Magento-provided patches included in the `{{site.data.var.mcp}}` package.
+1. Apply custom patches in the `/m2-hotfixes` directory in alphabetical order by patch name.
 
-{:.procedure}
-To use {{ site.data.var.mcp }} as a stand-alone package:
+You can also apply patches manually to test them in a local environment before you deploy to Staging and Production.
 
-1. Add the {{site.data.var.mcp}} package to your `composer.json` file.
+## Prerequisite
 
-    ```bash
-    composer require magento/magento-cloud-patches
-    ```
+{% include cloud/note-upgrade.md %}
+
+## Apply patches manually
+
+You can apply patches manually in a local environment and test them before you deploy to Staging and Production.
```

Comment:
The same as in my first comment - it sounds like patches won't be applied on Integration, only on Staging and Production